<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="setup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>601</height>
     </rect>
    </property>
    <property name="title">
     <string>Configuração</string>
    </property>
    <widget class="QTextEdit" name="t_tf2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>390</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="l_tec1">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>40</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TEC1</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="t_tec2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="t_tf1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="t_ts1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="t_ts2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>310</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="l_ts1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TS1</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_tef2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TEF2</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_tf2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TF2</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_tec2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TEC2</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="t_tef2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>350</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="l_ts2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TS2</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="t_tef1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="t_tec1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>151</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="l_tef1">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>160</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TEF1</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_tf1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TF1</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_tempo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tempo de simulação</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="t_tempo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="l_path">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Path do relatorio</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="t_path">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="run">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>560</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Executar</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_tfe1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TFE1</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="t_tfe1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="l_tfe2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TF1</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="t_tfe2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>270</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>301</width>
      <height>761</height>
     </rect>
    </property>
    <property name="title">
     <string>Relatorio</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Número Médio de Entidades nas Filas</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Taxa Média de Ocupação dos Servidores</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tempo Médio de uma Entidade na Fila</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tempo Médio no Sistema</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Contador</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Entraram no Sistema</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sairam do Sistema</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Permanecem no Sistema</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>320</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>380</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>360</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>420</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>440</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Falhas</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>540</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tempo em falha</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>480</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Numero de falhas</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>600</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>%Tempo em falha</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>520</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Servidor 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>500</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Servidor 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>580</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Servidor 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>560</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Servidor 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>640</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Servidor 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>620</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Servidor 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Servidor 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Servidor 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Servidor 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Servidor 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>660</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Contagem de trocas</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>740</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Tempo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>680</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Entidades perdidas</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>720</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>700</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo 1</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Execução</string>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>104</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Avançar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Terminar simulação</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cancelar simulação</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
